package/resource_monitor/resource_monitor/src/query_process_file_descriptor.c

```diff
@@ -152,8 +152,8 @@ static bool process_fd_entry(resq_state_t *state, const char *file_str, const ch
 static bool parse_fd_line(resq_state_t *state, const char *line)
 {
   char id_str[16] = {0};
-  char exe_str[32] = {0};
-  char file_str[64] = {0};
+  char exe_str[64] = {0};
+  char file_str[128] = {0};
```

Comment:
This should be NAME_MAX and PATH_MAX, respectively.  See https://www.gnu.org/software/libc/manual/html_node/Limits-for-Files.html.